src/main/scala/cromwell/binding/values/WdlString.scala

```diff
@@ -4,7 +4,13 @@ import cromwell.binding.types.WdlStringType
 import org.apache.commons.lang3.StringEscapeUtils
 import scala.util.{Success, Try}
 
-case class WdlString(value: String) extends WdlPrimitive {
+trait WdlStringLike {
+  def value: String
+  def isGcsUrl: Boolean = value.startsWith("gs://")
+  def isUriWithProtocol: Boolean = "^[a-zA-Z]+://".r.findFirstIn(value).nonEmpty
```

Comment:
ha ok got it it's because of the regex, I thought it was meant to be functionally different from a `startsWith` and was wondering why.